sapp/ui/frontend/src/index.js

```diff
@@ -27,9 +27,16 @@ import 'antd/dist/antd.css';
 const {Header, Content, Footer} = Layout;
 const {Text} = Typography;
 
+let uri = '/graphql';
+if (process.env.NODE_ENV === "development") {
+  uri = 'http://localhost:5000/graphql';
+}
 const client = new ApolloClient({
-  uri: '/graphql',
+  uri: uri,
   cache: new InMemoryCache(),
+  fetchOptions: {
+    mode: 'no-cors',
```

Comment:
Why do we need this? If it is only needed for development can we just add this option in the development and keep the production without it?

Reply:
We don't need this!